### ISSUE TITLE
About how to obtain human/object bbox and about ROI features 

### ISSUE THREAD

**Issue by wlsh1up** [RomeroBarata/human_object_interaction#14]
Hi,
Thank you for your paper. I'm very interested in it. and I am very new to this filed so I hope my questions are not irrelevant.
How to obtain the bbox of each frame object in training set and test set?  Is this provided by the dataset or detected by other models?  

I found that the objects bounding boxes in some pictures are [nan, nan, nan, nan], But the corresponding ROI feature you provided is not non. Why?

Thanks.
